src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -397,50 +371,9 @@ public boolean configure(final StaplerRequest req, final JSONObject formData) th
             this.setEmitSystemEvents(formData.getBoolean("emitSystemEvents"));
             this.setCollectBuildLogs(formData.getBoolean("collectBuildLogs"));
 
-            if(!validateTargetApiURL(this.getTargetApiURL())){
-                logger.severe("The field (targetApiURL) must be configured in the form <http|https>://<url>/");
-                return false;
-            }
-            if(!validateTargetLogIntakeURL(this.getTargetLogIntakeURL())){
-                logger.severe("The field (targetLogIntakeURL) must be configured in the form <http|https>://<url>/");
-                return false;
-            }
-            if(!validateTargetPort(portStr)){
-                logger.severe("Invalid Port");
-                return false;
-            }
-            if(!validateTargetLogCollectionPort(logCollectionPortStr)){
-                logger.severe("Invalid log collection port");
-                return false;
-            }
             if(StringUtils.isNotBlank(this.getHostname()) && !DatadogUtilities.isValidHostname(this.getHostname())){
                 throw new FormException("Your hostname is invalid, likely because it violates the format set in RFC 1123", "hostname");
             }
-            // Run connection validation if applicable
-            if(DatadogClient.ClientType.HTTP.name().equals(this.getReportWith()) &&
-                    !DatadogHttpClient.validateDefaultIntakeConnection(this.getTargetApiURL(), this.getTargetApiKey())){
-
-                // If a client instance exist we set the connectionBroken attribute to true.
-                DatadogClient client = DatadogHttpClient.getInstance(this.getTargetApiURL(),
-                        this.getTargetLogIntakeURL(), this.getTargetApiKey());
-                if(client != null){
-                    client.setDefaultIntakeConnectionBroken(true);
-                }
-                logger.severe("Connection broken, please double check both your API URL and Key");
-                return false;
-            }
-            if(DatadogClient.ClientType.HTTP.name().equals(this.getReportWith()) &&
-                    !DatadogHttpClient.validateLogIntakeConnection(this.getTargetLogIntakeURL(), this.getTargetApiKey())){
-
-                // If a client instance exist we set the connectionBroken attribute to true.
-                DatadogClient client = DatadogHttpClient.getInstance(this.getTargetApiURL(),
-                        this.getTargetLogIntakeURL(), this.getTargetApiKey());
-                if(client != null){
-                    client.setLogIntakeConnectionBroken(true);
-                }
-                logger.severe("Connection broken, please double check both your Log Intake URL and Key");
-                return false;
-            }
 
             //When form is saved....
             DatadogClient client = ClientFactory.getClient(DatadogClient.ClientType.valueOf(this.getReportWith()),
```

Comment:
I would try to see if we can scope down the try/except around that block. To make it clear that we except it to fail if the configuration looks valid but is not reachable for example.

Reply:
Sure, I can look into it!